macros/models/_project/oneinch/oneinch_calls_transfers_macro.sql

```diff
@@ -0,0 +1,177 @@
+{% macro 
+    oneinch_calls_transfers_macro(
+        blockchain,
+        project_start_date_str
+    ) 
+%}
+
+{% set project_start_date = "timestamp '" + project_start_date_str + "'" %} 
```

Comment:
@grkhr why don't you try editing the project start date on each blockchain specific spell, to just ~one month or so of data. that way we can try to see them run through fine. we can continue to expand the historical data back, if successful, until we hit the issue again. at least at that point, we'd have a sample of data get through and get some confidence to try and publish to prod.

Reply:
I've tried that [already](https://github.com/duneanalytics/spellbook/actions/runs/6229120776/job/16907069711) and it passed well :)